src/toolbox/ngm-toolbox.ts

```diff
@@ -26,7 +27,7 @@ import {showSnackbarInfo} from '../notifications';
 @customElement('ngm-tools')
 export class NgmToolbox extends LitElementI18n {
   @property({type: Boolean}) toolsHidden = true;
-  @state() activeTool: 'draw' | 'slicing' | 'gst' | undefined;
+  @property() activeTool: 'draw' | 'slicing' | 'gst' | 'data-download' | undefined;
```

Comment:
why `@property`?

Reply:
reverted in https://github.com/swissgeol/ngm/pull/773/commits/fdadeb903bf1ac1decea4be8ded6be9bdbd6a591